### PR TITLE
DENG-518 generated charts in pdf

### PR DESCRIPTION
**Pull request by wwyc**
[[DENG-518](https://mozilla-hub.atlassian.net/browse/DENG-518)]
Scatter plots are generated and saved into pdfs from analysis results.  Showing only top 10 results for now.

### CODE REVIEW

File: analysis/reports/generator.py [hunk 1]
```diff
@@ -68,3 +76,64 @@ def build_pdf_report(self) -> str:
             verbose=True,
         )
         return self.output_pdf
+
+    def build_pdf_charts(self):
+
+        output_pdf = PdfPages(
+            os.path.join(
+                self.output_dir, "generated_charts", self.filename_base + "_charts" + ".pdf"
```
Comment (gleonard-m) [inline analysis/reports/generator.py]: Does `generated_charts` will need to be created prior?

**Review by gleonard-m — COMMENTED** [review]
Don't forget to add the following to `pyproject.toml`
- matplotlib
- seaborn
- adjustText

File: analysis/reports/generator.py (hunk 1 above)
Comment (wwyc) [reply analysis/reports/generator.py]: can you explain more on this?

Comment (gleonard-m) [reply analysis/reports/generator.py]: I pulled your branch and when I ran it I got an error that `generated_charts` did not exist. I had to create it manually for the code to run successfully.

Comment (wwyc) [reply analysis/reports/generator.py]: thanks for catching that!  I added a check for the folder and create if not exists.

**Pull request closed without merging**